crates/iota-graphql-rpc/src/server/builder.rs

```diff
@@ -586,6 +580,35 @@ async fn graphql_handler(
     (extensions, result.into())
 }
 
+pub(crate) fn get_write_api<'ctx>(
+    ctx: &'ctx Context<'_>,
+) -> Result<&'ctx OptimisticWriteApi, Error> {
+    let write_api: &Option<OptimisticWriteApi> = ctx
+        .data()
+        .map_err(|_| Error::Internal("Unable to get node execution interface".to_string()))?;
+    write_api
+        .as_ref()
+        .ok_or_else(|| Error::Internal("Node execution interface not initialized".to_string()))
+}
+
+fn get_json_rpc_client(rpc_client_url: &str) -> Result<HttpClient, Error> {
```

Comment:
nit: Maybe we could rename it to: `new_json_rpc_client` or `build_json_rpc_client`? The `get_` prefix is a bit misleading since we're not getting an existing value.

Reply:
Done with 8d1cd55e9a